apps/native-component-list/src/screens/ModulesCore/ModulesCoreScreen.tsx

```diff
@@ -16,6 +16,13 @@ export const ModulesCoreScreens = [
       return optionalRequire(() => require('./ExpoModulesScreen'));
     },
   },
+  {
+    name: 'Benchmarks',
+    route: 'modulescore/benchmarks',
+    getComponent() {
+      return optionalRequire(() => require('./ModulesBenchmarksScreen'));
+    },
+  },
```

Comment:
this is now ios only, maybe we should guard the screen for `process.env.EXPO.OS === 'ios'`. also this screen should be excluded from expo-go.